osu.Game.Rulesets.Taiko/Difficulty/Skills/Reading.cs

```diff
@@ -29,15 +29,13 @@ public Reading(Mod[] mods)
 
         protected override double StrainValueOf(DifficultyHitObject current)
         {
-            if (current is not TaikoDifficultyHitObject taikoObject)
-                return 0.0;
-
-            // Drum Rolls and Swells are exempt.
             if (current.BaseObject is not Hit)
```

Comment:
Add the comment from before back as this still applies.

// Drum Rolls and Swells are exempt.